<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hall</class>
 <widget class="QDialog" name="hall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>在</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_5">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>天内停车费即将到期人员</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QTableView" name="tableView">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="lineWidth">
                <number>1</number>
               </property>
               <property name="gridStyle">
                <enum>Qt::CustomDashLine</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>数据筛选</string>
               </property>
               <widget class="QLabel" name="label_4">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>20</y>
                  <width>54</width>
                  <height>12</height>
                 </rect>
                </property>
                <property name="text">
                 <string>楼</string>
                </property>
               </widget>
               <widget class="QComboBox" name="comboBox">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>40</y>
                  <width>41</width>
                  <height>22</height>
                 </rect>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>导出到表</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>在31天内物业费即将到期人员</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QTableView" name="tableView_2"/>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>数据筛选</string>
               </property>
               <widget class="QLabel" name="label_3">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>20</y>
                  <width>54</width>
                  <height>12</height>
                 </rect>
                </property>
                <property name="text">
                 <string>楼</string>
                </property>
               </widget>
               <widget class="QComboBox" name="comboBox_2">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>40</y>
                  <width>41</width>
                  <height>22</height>
                 </rect>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>导出到表</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>请选择操作</string>
         </property>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>81</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>物业费中心</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>100</y>
            <width>81</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>停车费中心</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>240</y>
            <width>81</width>
            <height>191</height>
           </rect>
          </property>
          <property name="text">
           <string>刷新大厅数据</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
